packages/compass-indexes/src/stores/store.js

```diff
@@ -48,12 +48,20 @@ const configureStore = (options = {}) => {
     const globalAppRegistry = options.globalAppRegistry;
     store.dispatch(globalAppRegistryActivated(globalAppRegistry));
 
-    globalAppRegistry
-      .getStore('DeploymentAwareness.WriteStateStore')
-      .listen((state) => {
-        store.dispatch(writeStateChanged(state.isWritable));
-        store.dispatch(getDescription(state.description));
-      });
+    const deploymentAwarenessStore = globalAppRegistry.getStore(
+      'DeploymentAwareness.WriteStateStore'
+    );
+
+    deploymentAwarenessStore.listen((state) => {
+      store.dispatch(writeStateChanged(state.isWritable));
+      store.dispatch(getDescription(state.description));
+    });
+
+    // Load the initial deployment awareness state.
+    store.dispatch(
+      writeStateChanged(deploymentAwarenessStore.state.isWritable)
+    );
+    store.dispatch(getDescription(deploymentAwarenessStore.state.description));
```

Comment:
Adding this initial loading. This was a bug before, the `deploymentAwarenessStore`'s state would be out of line in this package as this compass-indexes store is initialized at a time when the deploymentAwareness state is already set.
Will be a nice thing to tackle in state management improvements, not including in this pr as we probably want those to be more easily discussed.